hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -572,6 +580,51 @@ def test_broadcastable_shape_bounds_are_satisfied(shape, data):
     assert all(min_side <= s <= max_side for s in bshape)
 
 
+@settings(deadline=None, suppress_health_check=[HealthCheck.too_slow])
+@given(
+    inputs=st.integers(1, 4),
```

Comment:
Best if tests reflect the as-merged argument names, to save the sanity of any software archeologists (who need all the help they can get around here).  So switch `inputs` for `num_shapes` wherever it appears.